test/e2e/usersignup_test.go

```diff
@@ -504,3 +504,32 @@ func (s *userSignupIntegrationTest) createUserSignupVerificationRequiredAndAsser
 	require.Error(s.T(), err)
 	return userSignup
 }
+
+func (s *userSignupIntegrationTest) TestSkipSpaceCreation() {
+	// given
+	hostAwait := s.Host()
+	hostAwait.UpdateToolchainConfig(testconfig.AutomaticApproval().Enabled(true))
+
+	// when
+	userSignup, _ := NewSignupRequest(s.T(), s.Awaitilities).
+		Username("nospace").
+		Email("nospace@redhat.com").
+		NoSpace().
+		EnsureMUR().
```

Comment:
`EnsureMUR()` automatically checks all resources to be provisioned for UserSignup - instead of it, you should just wait until MUR is created, otherwise, it would wail that it cannot find the Space
```suggestion
		WaitForMUR()
```